### PR TITLE
Expanded LiveScript toggles to include ! and added ! to JSX attribute toggle shorthand

### PR DESCRIPTION
**Pull request by STRd6**

### CODE REVIEW

**Review by edemaine — COMMENTED** [review]
This seems fine to me, aside from lack of documentation. Just want to point out:

* No existing language does this (but JSX open issue does)
* Redundant with `-`
* Asymmetric with `+`
* `{!foo}` (which means `{foo: false}`) is quite asymmetric/different from `{foo}` (which means `{foo: foo}`)

**Comment by edemaine** [comment]
Hmm, maybe I'd be happier with the symmetry if we also added `!!` for the true version??

**Comment by STRd6** [comment]
> Hmm, maybe I'd be happier with the symmetry if we also added `!!` for the true version??

A possibility for sure.

`{¡foo}` -> `{foo: true}` 🙃

File: civet.dev/cheatsheet.md [hunk 1]
```diff
@@ -41,11 +41,21 @@ c .= 10
...
+</Playground>
+
+You can use a flagging shorthand inspired by [LiveScript](https://livescript.net/#literals-objects).
+
+<Playground>
+config := {
+  +debug
+  -live
+  !verbose
+}
```
Comment (edemaine) [inline civet.dev/cheatsheet.md:58]: This actually duplicates an example below; could you remove that one?

**Pull request merged**